src/main/java/microsoft/exchange/webservices/data/autodiscover/AlternateMailbox.java

```diff
@@ -164,8 +177,44 @@ public String getServer() {
    *
    * @param server the new server
    */
-  protected void setServer(String server) {
+  protected void setServer(final String server) {
     this.server = server;
   }
 
+  /**
+   * Gets the SMTP address.
+   *
+   * @return
+   */
+  public String getSmtpAddress() {
```

Comment:
I can add documentation on the properties or the class.

The difference is that the OwnerSmtpAddress is the SMTP address of owner of the alternate mailbox, and the SmtpAddress belongs to the mailbox itself.

The SmtpAddress is only set if it is available for the given Type e.g. Delegate has an SmtpAddress set Archive not.